spec/features/user_votes_spec.rb

```diff
@@ -20,14 +21,18 @@
 
       click_button "Log in"
 
-      visit station_path(review.station)
+      visit station_path(@review.station)
     end
 
-    scenario 'User votes on review' do
-      # figure out voting mechanism on form
+    scenario 'User votes on review', focus: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.